sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -1219,11 +1240,12 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     const requestedOffering = await this.getOfferingToRequestChange(
       offeringId,
       OfferingStatus.ChangeAwaitingApproval,
+      { isIncludeInActiveProgram: false },
```

Comment:
Assess offering change should not be impacted by deactivated program. 

Once an offering change is requested, as per the business ministry can action on it.

@andrewsignori-aot @andrepestana-aot @sh16011993

Reply:
If the offering request started before the program was deactivated the Ministry should be able to approve.